README.md

```diff
@@ -93,6 +93,10 @@ oc apply -f config/samples/example-fedora.yaml
 ## Uninstall
 
 ### Openshift
+Prior to deletion of your kataconfig, it is expected that all existing pods that use kata runtime have been removed.
+The `oc delete` operation will fail if there are pods in the cluster that are still referencing the kata runtime.
```

Comment:
But it won't fail in the sense that the command is aborted, it will sit there waiting for the pods to be deleted. An error message will only show up in the log. Maybe it's worth making that more clear in the README until we fix it? A fix could be to put the error into the kataconfig status, but that's for another PR.

Reply:
@pruan-rht could you change that last sentence to say something like "The 'oc delete' operatorion will block if there are pods in the cluster that are still using the kata runtime until those pods have been deleted"